ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -117,5 +117,20 @@
     "CaseEventID": "handleSupplementaryEvidence",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "attachScannedDocs",
+    "UserRole": "caseworker-publiclaw-bulkscan",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "attachScannedDocs",
+    "UserRole": "caseworker-publiclaw-systemupdate",
```

Comment:
@tomaszpowroznik do we need to update this to be caseworker-publiclaw-bulkscansystemupdate?

Reply:
Yes should have `caseworker-publiclaw-bulkscansystemupdate` so automatic supplementary evideence handling can happen